### DISCUSSION TITLE
How to send a monospaced message?

### DISCUSSION THREAD

**Discussion by Weckerlin** [windkh/node-red-contrib-telegrambot#360] [Q&A] [answered]
I wanted my bot to send a message in monospace format, how do I do that?

**Comment by windkh** [comment, marked as answer]
Try out setting parse_mode to Markdown or MarkdownV2

**Reply by Weckerlin** [reply]
It worked, thank you very much!

`msg.payload.options = { parse_mode: "Markdown" };`
